src/libraries/Common/src/System/IO/Compression/Archiving.Utils.cs

```diff
@@ -7,9 +7,11 @@
 
 namespace System.IO.Compression
 {
-    internal static partial class ZipFileUtils
+    internal static partial class ArchivingUtils
     {
-        // Per the .ZIP File Format Specification 4.4.17.1 all slashes should be forward slashes
+        // To ensure tar files remain compatible with Unix,
+        // and per the ZIP File Format Specification 4.4.17.1,
```

Comment:
Slight comment adjustment since this class is now being used by both zip and tar APIs.